rules/nextstrain_exports.smk

```diff
@@ -256,38 +256,39 @@ except:
     # means that the Snakefile won't crash.
     deploy_origin = "by an unknown identity"
 
-rule deploy_to_staging:
-    input:
-        *rules.all_regions.input
-    params:
-        slack_message = f"Deployed <https://nextstrain.org/staging/ncov|nextstrain.org/staging/ncov> {deploy_origin}",
-        s3_staging_url = config["s3_staging_url"]
-    conda: config["conda_environment"]
-    shell:
-        """
-        nextstrain deploy {params.s3_staging_url:q} {input:q}
+if config["connect_to_s3"]:

```

Comment:
Perhaps if not `config["connect_to_s3"]`, we don't want to do any of the deployment and error handling setup above this line.